components/ScheduleCard.js

```diff
@@ -21,6 +30,19 @@ export default class ScheduleCard extends Component<Props> {
             position: "absolute", alignSelf: "flex-end",
             flex: 1, padding: 8
           }}>
+            <Text>Subscribe to Event?</Text>
+            <Switch
+            value={this.state.switchValue}
+            onValueChange ={(switchValue)=> {
+              this.setState({switchValue});
+              if (switchValue) OneSignal.sendTag(this.props.title + "eventid", "subscribed");
```

Comment:
ooh damn tags as in onesignal tags? 

We were going to use tags as an in-hackgt concept, defined by our CMS, used across apps. Since other interfaces are not be able to subscribe/unsubscribe from tags/can't access this api, this feels premature - the onesignal id should be handled through buzzer, or ground truth. concerns here - feel free to debate me here

Also note the misnomer with individual events being tags here - these are like starring events. My opinion is that you should maintain a `React.Context` with a Set of these events and verify your event is or is not in the event. Notifications are filtered if they're about a particular eventid (followed) or a tag that meha handles - we can avoid the dependency on onesignal api. In the future, if this interest info per user is stored more centrally, we can maybe filter onesignal side.

Also, what exactly is this tag syntax doing? Subscribing and unsubscribing from a particular eventid? As a "boolean" string?

Reply:
Wait I'm a bit confused. If I'm not supposed to use onesignal tags, how did you want to filter the notifications appside?